package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "subway",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
```

Comment:
Why this file is main? It is a start point of your application. Usually index.html or index.js